app/views/admin_posts/show.html.erb

```diff
@@ -1,64 +1,38 @@
-<div class="admin home">
+<div class="news admin home">
   <div class="primary header module">
     <!--Descriptive page name, messages and instructions-->
     <h2 class="heading"><%= link_to ts("AO3 News"), admin_posts_path %></h2>
     <div class="icon"></div>
     <!--/descriptions-->
     <!--subnav-->
-    <ul class="navigation actions" role="navigation">
-      <% if @previous_admin_post %>
-        <li><%= link_to("Previous Post", @previous_admin_post) %></li>
+    <div class="navigation actions module">
+      <% if logged_in_as_admin? %>
+        <h4 class="landmark heading"><%= ts("Admin Actions") %></h4>
+        <ul class="management actions" role="navigation">
```

Comment:
Put the Edit Post/Delete Post links up here since the old location no longer works.